packages/aws-rfdk/lib/deadline/lib/repository.ts

```diff
@@ -519,6 +558,23 @@ export class Repository extends Construct implements IRepository {
 
     this.version = props.version;
 
+    this.secretsManagementSettings = {
+      enabled: props.secretsManagementSettings?.enabled ?? true,
+      credentials: props.secretsManagementSettings?.credentials ??
+        ((props.secretsManagementSettings?.enabled ?? true) ? new Secret(this, 'SMAdminUser', {
+          description: 'Admin credentials for Secret Management',
+          generateSecretString: {
+            excludeCharacters: '\"$&\'()-/<>[\\]\`{|}',
+            includeSpace: false,
+            passwordLength: 24,
+            requireEachIncludedType: true,
+
+            generateStringKey: 'password',
+            secretStringTemplate: JSON.stringify({ username: 'admin' }),
```

Comment:
We can pull `admin` out into a `DEFAULT_SECRETS_MANAGEMENT_USERNAME` constant.

Reply:
It is our normal practice in RFDK to set username for admin credentials `admin` by default. User can always setup own username and password in custom secret.

Reply:
Yes that's fine to set it to `admin` by default, I just mean to pull it into a constant and refer to that here rather than hardcoding the value.